ratis-netty/src/main/java/org/apache/ratis/netty/server/NettyServerStreamRpc.java

```diff
@@ -51,27 +57,52 @@
   private RandomAccessFile stream;
   private FileChannel fileChannel;
   private File file = new File("client-data-stream");
+  private List<DataStreamClientImpl> clients = new ArrayList<>();
+  private List<DataStreamOutput> streams = new ArrayList<>();
```

Comment:
Maybe `clients` is enough, we do not need streams, it cause confused, when we need stream, we get it from `client.stream()`

Reply:
yeah this is a good point.

Each `client.stream()` will increase stream id, I am not sure if changing stream id matters. For example, if client sends 100 message, they will under the format of same stream id and 100 different message id. if we do `client.stream()` each time here, the messages forwarded to peers will become the format of different stream id, same message id (assume message id counts from 0 for each stream id), which seemed not right.

However, I am not sure when stream id for the forwarded steam should be updated within a `NettyServerStreamRpc`

Reply:
@szetszwo Could you help explain this ?